packages/nextjs/pages/builders/0x2F142f9F52af0845B7b8b1353E349Ca1Cb8EFE92.tsx

```diff
@@ -0,0 +1,92 @@
+import { useEffect, useState } from "react";
+import Link from "next/link";
+import { appChains } from "~~/services/web3/wagmiConnectors";
+
+export default function PapaWheelie() {
+  const [ensData, setEnsData] = useState<Record<string, string | undefined>>({});
+  const nameTitle = "P A P A W H E E L I E";
+  const ensName = "papawheelie.eth";
+
+  useEffect(() => {
+    const client = appChains.publicClient({
+      chainId: 1,
+    });
+    async function fetchEnsTexts() {
+      const keys = ["com.twitter", "bio", "com.discord", "com.youtube", "com.github", "avatar"];
+      try {
+        const texts = await Promise.all(
+          keys.map(key =>
+            client
+              .getEnsText({
+                name: "papawheelie.eth",
+                key: key,
+              })
+              .then(text => ({ key, text })),
+          ),
+        );
+
+        const newData = texts.reduce((acc, { key, text }) => {
+          acc[key] = text ?? undefined;
+          return acc;
+        }, {} as Record<string, string | undefined>);
+
+        setEnsData(newData);
+      } catch (error) {
+        console.error("Error fetching ENS texts:", error);
+      }
+    }
+
+    fetchEnsTexts();
+  }, []);
+
+  return (
+    <div className="flex h-lvh flex-grow items-center justify-center px-4">
+      <div className="mx-auto max-w-3xl rounded-md shadow-2xl shadow-indigo-500/40 md:shadow-indigo-500/40 md:shadow-2xl">
+        <div className="flex flex-col md:flex-row items-center items-stretch justify-center gap-4">
+          <div className="md:w-1/3 flex justify-center md:justify-start">
+            <img
+              className="rounded-full w-1/2 h-1/2 md:h-full md:w-full md:object-cover md:rounded-l-md md:rounded-r-none"
+              src={ensData["avatar"] ?? "https://i.imgur.com/UhV7H97.jpeg"}
+              alt={ensData["name"] ?? "Loading..."}
+            />
+          </div>
+          <div className="md:w-2/3 flex flex-col justify-between p-8">
+            <div className="text-center md:text-start divide-y divide-solid divide-slate-500/30">
+              <p className="my-0">{nameTitle}</p>
+              <span className="text-sm text-gray-500 dark:text-gray-400">{ensName}</span>
+            </div>
+            <div className="text-center md:text-left pb-2">
+              <p>{ensData["bio"] ?? "Loading..."}</p>
+            </div>
+            <div className="flex flex-wrap justify-between items-end">
+              <Link
```

Comment:
Maybe have the links with `_blank`? So they open in a new window.

Reply:
good call, added and tested.